.circleci/config.yml

```diff
@@ -281,143 +281,164 @@ defaults:
   # --------------------------------------------------------------------------
   # Base Image Templates
 
-  - base_ubuntu1604_clang: &base_ubuntu1604_clang
+  - base_archlinux: &base_archlinux
```

Comment:
I see you sorted the base dicts? Can you save that change for the follow-up refactor PR? The current PR's diff suddenly got harder to review because of this :)

In general I'd avoid adding bigger refactors in such PRs, especially when we know there's enough of such small tweaks to warrant a separate follow-up PR as in this case. It would be best if it had only just enough to get it done. We want to have this ready for the release so it should be as simple as possible. Cleaning up the config is a good thing, but we should do it in a separate PR, where it does not distract from actual functional changes. This way we can also give it enough attention to get it right without worrying that doing it properly will delay the release :)

Reply:
Sure. Yeah, I honestly was a bit in doubt if I should do it here or not, based on this comment: https://github.com/ethereum/solidity/pull/13973#discussion_r1108446308

But yeah, I moved it to the refactor branch and only  kept in this PR the addition of the base images for 20.04 with different resource classes.

Reply:
Ah, I see the confusion now. I meant just to keep the current structure, not sort them. Sorry for the ambiguity, I should have phrased that better. It does sound as if I asked you to sort them :)